# Topics
camera_topic: "/oak/rgb/image_raw"
camera_info: "/oak/rgb/camera_info"
lidar_topic: "/lidar_calibration_utils/lidar_points"

# Dynamic rqt_reconfigure default bounds
feature_extraction:
  x_min: -10.0
  x_max: 10.0
  y_min: -8.0
  y_max: 8.0
  z_min: -5.0
  z_max: 5.0
  
# Properties of chessboard calibration target
chessboard:
  pattern_size:
    height: 7
    width: 5  
  square_length: 95
  board_dimension:
    width: 600
    height: 800
  translation_error:
    x: 5
    y: 2